packages/prover/README.md

```diff
@@ -37,13 +35,12 @@ console.log({balance, address});
 You can also invoke the package as binary.
 
 ```bash
-npm -i g @lodestar/prover
+npm i g @lodestar/prover
 
 lodestar-prover start \
   --network sepolia \
-  --execution-rpc https://lodestar-sepoliarpc.chainsafe.io \
-  --mode rest \
-  --beacon-rpc https://lodestar-sepolia.chainsafe.io \
+  --execution-rpc-url https://lodestar-sepoliarpc.chainsafe.io \
```

Comment:
should use camel case for CLI flags

```suggestion
  --executionRpcUrl https://lodestar-sepoliarpc.chainsafe.io \
```

Reply:
Both works. Do you mean that this is the preferred way?

Reply:
yargs by default supports snake-case but for consistency should use camelCase